packages/cli/src/commands/deploy/helper.ts

```diff
@@ -414,3 +418,37 @@ export const createFieldPlugin = async (
     )
   }
 }
+
+// Publish Logic
+
+type CheckPublish = (params: {
+  publish: undefined | boolean
+  skipPrompts: undefined | boolean
+}) => Promise<boolean>
+
+export const checkPublish: CheckPublish = async ({ publish, skipPrompts }) => {
+  if (skipPrompts === true) {
+    //NOTE: Publish true is the default value
+    return publish || true
```

Comment:
nit: Maybe here it could be simplified like that, Bibi? wdyt?

```suggestion
    //NOTE: Publish true is the default value
    return true
```

Reply:
Hey @demetriusfeijoo, if skipPrompts is true, it can still be the case that the --no-publish flag was defined. If so it should be taken into account. `|| true` is only for this case `yarn deploy --skipPrompts` where no publish flag is defined. 

WDYT?